lib/node_modules/@stdlib/array/complex128/test/test.find.js

```diff
@@ -0,0 +1,177 @@
+	t.end();
+});
+
+tape( 'attached to the prototype of the main export is a `find` method', function test( t ) {
+	t.strictEqual( hasOwnProp( Complex128Array.prototype, 'find' ), true, 'has property' );
+	t.strictEqual( isFunction( Complex128Array.prototype.find ), true, 'has method' );
+	t.end();
+});
+
+tape( 'the method throws an error if invoked with a `this` context which is not a complex number array instance', function test( t ) {
+	var values;
+	var arr;
+	var i;
+
+	arr = new Complex128Array( 5 );
+
+	values = [
+		'5',
+		5,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		{},
+		[],
+		function noop() {}
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			return arr.find.call( value, predicate );
+		};
+	}
+
+	function predicate( v ) {
+		return ( real( v ) > 0 && imag( v ) < 0 );
+	}
+});
+
+tape( 'the method throws an error if provided a first argument which is not a function', function test( t ) {
+	var values;
+	var arr;
+	var i;
+
+	arr = new Complex128Array( 10 );
+
+	values = [
+		'5',
+		3.14,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		{},
+		[]
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			return arr.find( value );
+		};
+	}
+});
+
+tape( 'the method returns `undefined` if provided an empty complex number array', function test( t ) {
+	var arr;
+	var z;
+
+	arr = new Complex128Array();
+	z = arr.find( predicate );
+
+	t.strictEqual( z, void 0, 'returns expected value' );
+	t.end();
+
+	function predicate() {
+		t.fail( 'should not be invoked' );
+	}
+});
+
+tape( 'the method returns the first element which passes a test', function test( t ) {
+	var arr;
+	var z;
+
+	arr = new Complex128Array( [ 1.0, 1.0, 2.0, -2.0 ] );
+	z = arr.find( predicate );
+
+	t.strictEqual( real( z ), 2.0, 'returns expected value' );
+	t.strictEqual( imag( z ), -2.0, 'returns expected value' );
+	t.end();
+
+	function predicate( v ) {
+		return ( real( v ) === -imag( v ) );
+	}
+});
+
+tape( 'the method returns `undefined` if all elements fail a test', function test( t ) {
+	var arr;
+	var z;
+
+	arr = new Complex128Array( [ 1.0, -1.0, 2.0, -2.0, 3.0, -3.0 ] );
+	z = arr.find( predicate );
+
+	t.strictEqual( z, void 0, 'returns expected value' );
+	t.end();
+
+	function predicate( v ) {
+		return ( real( v ) === imag( v ) );
+	}
+});
+
+tape( 'the method supports providing an execution context', function test( t ) {
+	var ctx;
+	var arr;
+	var z;
+
+	ctx = {
+		'count': 0
+	};
+	arr = new Complex128Array( [ 1.0, -1.0, 2.0, -2.0, 3.0, 3.0 ] );
+	z = arr.find( predicate, ctx );
+
+	t.strictEqual( real( z ), 3, 'returns expected value');
+	t.strictEqual( imag( z ), 3, 'returns expected value');
```

Comment:
```suggestion
	t.strictEqual( real( z ), 3.0, 'returns expected value');
	t.strictEqual( imag( z ), 3.0, 'returns expected value');
```

In order to match the values provided in L164. When a value will always be integral, we omit the decimal. However, if a value may assume any fractional value (integral or otherwise), we include the decimal. E.g., a count should always be integral, while a complex number component may be integral or any finite fractional value.